layers/meta-tegra-support/recipes-devtools/vpi/vpi1-samples_1.0.15.bb

```diff
@@ -0,0 +1,89 @@
+SUMMARY = "NVIDIA VPI command-line sample applications"
+HOMEPAGE = "https://developer.nvidia.com/embedded/vpi"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://opt/nvidia/vpi1/samples/assets/LICENSE;md5=6b5f633fc3acaabf21035790a05b1c71"
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+inherit l4t_deb_pkgfeed cuda
+
+SRC_COMMON_DEBS = "\
+    vpi1-samples_${PV}_arm64.deb;subdir=vpi1-samples \
+"
+SRC_URI[sha256sum] = "98ea137962cb4bbfb1a778faf1cc3b19e27bbd0473aa7ab21cf9fde3a0469d9b"
+
+SRC_URI_append = " file://0001-06-klt_tracker-minor-update-in-usage.patch"
+
+VPI_SAMPLES_PATH = "opt/nvidia/vpi1/samples"
+VPI_SAMPLES ?= " \
+    ${VPI_SAMPLES_PATH}/01-convolve_2d \
+    ${VPI_SAMPLES_PATH}/02-stereo_disparity \
+    ${VPI_SAMPLES_PATH}/03-harris_corners \
+    ${VPI_SAMPLES_PATH}/04-rescale \
+    ${VPI_SAMPLES_PATH}/05-timing \
+    ${VPI_SAMPLES_PATH}/06-klt_tracker \
+    ${VPI_SAMPLES_PATH}/07-fft \
+    ${VPI_SAMPLES_PATH}/09-tnr \
+    ${VPI_SAMPLES_PATH}/10-perspwarp \
+    ${VPI_SAMPLES_PATH}/11-fisheye \
+    ${VPI_SAMPLES_PATH}/12-optflow_lk \
+"
```

Comment:
I think the whole recipe would be simpler if we add a top-level `CMakeLists.txt` file to drive the build, and then just inherit `cmake`.